articles/virtual-machines/workloads/sap/deployment-guide.md

```diff
@@ -997,6 +995,9 @@ NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI mus
 
    1. Install Azure CLI AEM Extension
```

Comment:
1. Install Azure CLI AEM Extension. Make sure to use at least version 0.2.2 or higher.